docs/sources/write/tooling-and-workflows/_index.md

```diff
@@ -376,3 +376,16 @@ GitHub CLI simplifies some git workflows when working with GitHub repositories.
 ### Install GitHub CLI
 
 To install the GitHub CLI tool, refer to [Installation](https://github.com/cli/cli#installation).
+
+### Checkout a fork
```

Comment:
```suggestion
### Check out a fork
```

Reply:
These are interesting nits. The `git` and `gh` subcommand is `checkout` rather than `check-out` which implies to me that it is one word and not two. Another subcommand `cherry-pick` has the dash which implies to me that it is two words.

Reply:
When used as a verb, it's "check out". As a noun or adjective, which is how it's used in the GH command, it's "checkout".

Reply:
Nice that makes sense and is also what is present in the `git` docs (https://git-scm.com/docs/git-checkout).

Reply:
👍